src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplateField/PhrasalTemplateRenderField.tsx

```diff
@@ -0,0 +1,128 @@
+import { useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { useEffect } from 'react';
+
+import { useCustomFormContext } from 'modules/Builder/hooks';
+import { StyledMdPreview } from 'modules/Builder/components/ItemFlowSelectController/StyledMdPreview/StyledMdPreview.styles';
+import { Svg } from 'shared/components';
+import { applet } from 'shared/state/Applet';
+import { InputController, SelectController } from 'shared/components/FormComponents';
+import {
+  StyledBodyLarge,
+  StyledFlexTopCenter,
+  StyledLabelLarge,
+  theme,
+  variables,
+} from 'shared/styles';
+
+import { StyledLineBreak } from './PhrasalTemplateField.styles';
+import { PhrasalTemplateFieldProps } from './PhrasalTemplateField.types';
+
+export function RenderedField({
+  name = '',
+  responseOptions = [],
+  type = 'sentence',
+  ...otherProps
+}: Omit<PhrasalTemplateFieldProps, 'canRemove' | 'onRemove'>) {
+  const { t } = useTranslation('app');
+  const params = useParams();
+  const { control, getValues, setValue } = useCustomFormContext();
+  const fieldValue = getValues(name as string);
```

Comment:
Apparently `fieldValue` can be undefined here, so when `fieldValue.itemName` is not using optional chaining on line 33, the site crashes after publishing the applet:

https://github.com/user-attachments/assets/89f7780c-b435-4cc2-9b95-5ef49009781a

The reason TypeScript isn't catching this is because `getValues()` is returning `any`. Tightening up the typing here would be ideal, however using `PhrasalTemplateField` as below causes other TS issues the code as-is, because of variable prop presence (depending on `fieldValue.type`):

```suggestion
  const fieldValue = getValues(name as string) as undefined | PhrasalTemplateField;
```

To deal with those TS errors might require some minor refactoring, so instead of this suggestion ☝🏻, for now best to just ensure that when accessing `fieldValue.itemName` below, we use optional chaining as needed: `fieldValue?.itemName`